Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:2-alpine
```

Comment:
Do we use the dockerfile?

Also Python2?

Reply:
Forgot to remove the Dockerfile. Latest commit has it removed.